.github/workflows/conan.yml

```diff
@@ -40,6 +40,63 @@ jobs:
           github.event_name == 'schedule' &&
           steps.git_info.outputs.current_commit == steps.last_successful_commit.outputs.commit-hash
 
+  build_windows:
+    if: github.repository_owner == 'viamrobotics'
+    needs: [prepare]
+    runs-on: windows-latest
+    strategy:
+      fail-fast: false
+      matrix:
+        include:
+          - target: x86_64-windows
+            platform: windows_x86_64
+    steps:
+      - name: Checkout Code
+        uses: actions/checkout@v4
+
+      - name: Install cmake
+        uses: ssrobins/install-cmake@v1
+        with:
+          version: 3.27.2
+
+      - name: Install python
+        uses: actions/setup-python@v5
+        with:
+          python-version: '3.12'
+
+      - name: Install Scoop
+        uses: MinoruSekine/setup-scoop@v4.0.1
+
+      - name: install buf 'n stuff
+        run: scoop install buf ninja
```

Comment:
Do you still need any of this if you have choco?

Reply:
unfortunately I think so. possibly `choco` can take care of `ninja` for us but I was unable to install buf with anything except `scoop` (in fact, the official instructions for `buf` are to use `scoop`).

Reply:
I'm a little puzzled about this, because as far as I know I was able to build the C++ SDK as part of the windows tflite build without scoop, or many of these other steps.

Reply:
Also, this page, describing `windows-latest` claims that `cmake`, `python`, and `choco` come pre-installed:

https://github.com/actions/runner-images/blob/main/images/windows/Windows2022-Readme.md

Reply:
Oh: I made it so the C++ SDK _downloads_ buf. So you shouldn't need to install it at all!:

https://github.com/viamrobotics/viam-cpp-sdk/blob/main/src/viam/api/CMakeLists.txt#L64-L94